### PR TITLE
linux/[a-g]: add more information link

### PR DESCRIPTION
**Pull request by Stig124**
Partial fix for #6062 on the linux side of pages, with commits notarized by letters
- [x] The page (if new), does not already exist in the repo.
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/add-apt-repository.md
Comment (marchersimon) [inline pages/linux/add-apt-repository.md:4]:
```suggestion
> More information: <https://manned.org/apt-add-repository>.
```
Typically we'd use a man page like <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.html>, but <https://manned.org/apt-add-repository> seems to be more extensive here.

File: pages/linux/addr2line.md
Comment (marchersimon) [inline pages/linux/addr2line.md:4]:
```suggestion
> More information: <https://manned.org/addr2line>.
```
<https://www.gnu.org/software/binutils/> doesn't really give any information on the command.

File: pages/linux/amixer.md
Comment (marchersimon) [inline pages/linux/amixer.md:4]:
```suggestion
> More information: <https://manned.org/amixer>.
```
If the command isn't Arch specific, we generally prefer <https://www.manned.org>

File: pages/linux/as.md
Comment (marchersimon) [inline pages/linux/as.md:5]:
```suggestion
> More information: <https://manned.org/as>.
```
Same as with `addr2line`

File: pages/linux/beep.md
Comment (marchersimon) [inline pages/linux/beep.md:4]:
```suggestion
> More information: <https://github.com/spkr-beep/beep>.
```

File: pages/linux/blkdiscard.md
Comment (marchersimon) [inline pages/linux/blkdiscard.md:4]:
```suggestion
> More information: <https://manned.org/blkdiscard>.
```
The GitHub README doesn't provide any info on the command itself.

File: pages/linux/blkid.md
Comment (marchersimon) [inline pages/linux/blkid.md:4]:
```suggestion
> More information: <https://manned.org/blkid>.
```
Same here.

File: pages/linux/brctl.md
Comment (marchersimon) [inline pages/linux/brctl.md:4]:
```suggestion
> More information: <https://manned.org/brctl>.
```
I can't find any reference to `brctl` on this site.

File: pages/linux/btrfs.md
Comment (marchersimon) [inline pages/linux/btrfs.md:4]:
```suggestion
> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs>.
```
I think this would be more fitting.

File: pages/linux/cal.md
Comment (marchersimon) [inline pages/linux/cal.md:4]:
```suggestion
> More information: <https://manned.org/cal>.
```
Same as with the other `util-linux` commands.

File: pages/linux/calc.md
Comment (marchersimon) [inline pages/linux/calc.md:4]:
```suggestion
> More information: <https://github.com/lcn2/calc>.
```
How about that? I'm not a fan of the original website since it's `http`.

File: pages/linux/chage.md
Comment (marchersimon) [inline pages/linux/chage.md:4]:
```suggestion
> More information: <https://manned.org/chage>.
```
The repository isn't very useful to find information.

**Review by marchersimon — CHANGES_REQUESTED** [review]
Great work, thanks. I rewieved everything until `chage` for now, more is comming as soon as I have time for it.

**Comment by bl-ue** [comment]
I converted this PR to a draft — feel free to undo that once you're ready for review, @Stig124 :)

**Comment by bl-ue** [comment]
@Stig124 you'll probably want to use https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py, if you're not already.  It saves a lot of headache when dealing with the more infos 😅

**Comment by marchersimon** [comment]
Oh, and also please have a look at <https://lukwebsforge.github.io/tldri18n/#linux> to see if any pages have translations. Those should also be updated.

**Comment by Stig124** [comment]
> @Stig124 you'll probably want to use https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py, if you're not already. It saves a lot of headache when dealing with the more infos sweat_smile

Oh thanks

**Comment by Stig124** [comment]
And that script is broken

**Comment by Stig124** [comment]
```
scripts/set-more-info-link.py: line 7: labels: command not found
scripts/set-more-info-link.py: line 8: en:: command not found
scripts/set-more-info-link.py: line 9: bs:: command not found
scripts/set-more-info-link.py: line 10: da:: command not found
scripts/set-more-info-link.py: line 11: de:: command not found
scripts/set-more-info-link.py: line 12: es:: command not found
scripts/set-more-info-link.py: line 13: fa:: command not found
scripts/set-more-info-link.py: line 15: fr:: command not found
scripts/set-more-info-link.py: line 43: syntax error near unexpected token `('
scripts/set-more-info-link.py: line 43: `    f = os.path.normpath(__file__)'
```

**Comment by Stig124** [comment]
If I have time I will fix that

**Comment by marchersimon** [comment]
@bl-ue in cases like this it probably would be easier to first add all links to the English pages and one we agree on all of them just run a script to automatically update the links in all translated pages. Wdyt?

**Comment by Stig124** [comment]
And that will be trivial

**Comment by marchersimon** [comment]
@Stig124 did you run the script with `./scripts/set-more-info-link.py` or `python scripts/set-more-info-link.py`?

**Comment by bl-ue** [comment]
@Stig124
That looks to me like you're running like `./set-more-info-link.py`, which won't work because it's not a bash script.  Run it like this:
```
python3 scripts/set-more-info-link.py -p linux/command.md "https://example.com"
```

**Comment by bl-ue** [comment]
>@bl-ue in cases like this it probably would be easier to first add all links to the English pages and one we agree on all of them just run a script to automatically update the links in all translated pages. Wdyt?

@marchersimon Yeah, that sounds like a good idea. 👍🏻

**Comment by marchersimon** [comment]
@bl-ue why don't we just add `#!/usr/bin/env python` in the first line of the script to make it runnable with `./`?

**Comment by Stig124** [comment]
> @Stig124
> That looks to me like you're running like `./set-more-info-link.py`, which won't work because it's not a bash script. Run it like this:
> 
> ```
> python3 scripts/set-more-info-link.py -p linux/command.md "https://example.com"
> ```

Why isn't the shebang set though?

**Comment by bl-ue** [comment]
@marchersimon that's a great idea.

@Stig124 simply because I've never though of it.  Go ahead and open a PR to add it, if you'd like :).   Use `/usr/bin/env python3` though (not `python`).

**Comment by Stig124** [comment]
I just f* up my branches on my fork, can't do the fix now

**Comment by bl-ue** [comment]
Why?  I don't think so.  Just open a PR for the https://github.com/Stig124/tldr/tree/sfix2 branch.

**Comment by Stig124** [comment]
It has all of the link commits, so I'm unable

**Comment by Stig124** [comment]
I put the fix in this commit

**Comment by bl-ue** [comment]
Ah, I see.  Just checkout `main`, create a new branch, re-make the shebang change, PR.
```
git checkout main
git checkout -b set-more-info-link-py-shebang

<make changes ...>

git add scripts/set-more-info-link.py
git commit -m "set-more-info-link.py: add shebang"
git push -u origin set-more-info-link-py-shebang

<go to https://github.com/Stig124/tldr/pull/new/set-more-info-link-py-shebang>
```

**Comment by bl-ue** [comment]
>I put the fix in this commit

I'd prefer that go in its own PR. Just leave it in this PR though, and make a new PR per my instructions above.  Then, once it's merged, we'll update this PR, and that should essentially fix it all.

**Comment by Stig124** [comment]
> > I put the fix in this commit
> 
> I'd prefer that go in its own PR. Just leave it in this PR though, and make a new PR per my instructions above. Then, once it's merged, we'll update this PR, and that should essentially fix it all.

Done

**Comment by Stig124** [comment]
I'm done for today on that, feel free to review what I've done so far

**Comment by marchersimon** [comment]
@Stig124 I saw that you set the executable bit in https://github.com/tldr-pages/tldr/blob/3879b3c37f022d332ceba2838a534e818c25ccdc/scripts/send-to-bot.py. Nice change, but that shouldn't be done in this PR. Would you please make a seperate PR or remove it from here?

File: pages/linux/chkconfig.md
Comment (marchersimon) [inline pages/linux/chkconfig.md:4]:
```suggestion
> More information: <https://manned.org/chkconfig>.
```
I'm not sure if the Wikipedia article is very useful.

File: pages/linux/cmus.md
Comment (marchersimon) [inline pages/linux/cmus.md:5]:
```suggestion
> More information: <https://cmus.github.io>.
```
This saves us one redirect.

File: pages/linux/compgen.md
Comment (marchersimon) [inline pages/linux/compgen.md:4]:
```suggestion
> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-compgen>.
```
I think we should be a bit more specific here.

File: pages/linux/compose.md
Comment (marchersimon) [inline pages/linux/compose.md:5]:
```suggestion
> More information: <https://manned.org/compose>.
```
I think this would be more helpful.

**Comment by Stig124** [comment]
> @Stig124 I saw that you set the executable bit in https://github.com/tldr-pages/tldr/blob/3879b3c37f022d332ceba2838a534e818c25ccdc/scripts/send-to-bot.py. Nice change, but that shouldn't be done in this PR. Would you please make a seperate PR or remove it from here?

This is a mistake (see prior messages), this will be removed

File: pages/linux/debman.md
Comment (marchersimon) [inline pages/linux/debman.md:4]:
```suggestion
> More information: <https://manpages.debian.org/latest/debian-goodies/debman.1.html>.
```
We generally try to use man pages. (Also, when using `dabian.org` links we sholud use `latest` and remove the `en` locale.)

File: pages/linux/diff3.md
Comment (marchersimon) [inline pages/linux/diff3.md:4]:
```suggestion
> More information: <https://www.gnu.org/software/diffutils/manual/html_node/Invoking-diff3.html>.
```
How about this?

File: pages/linux/disown.md
Comment (marchersimon) [inline pages/linux/disown.md:5]:
```suggestion
> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-disown>.
```

File: pages/linux/dmenu.md
Comment (marchersimon) [inline pages/linux/dmenu.md:5]:
```suggestion
> More information: <https://manned.org/dmenu>.
```
Even though it's the official website, I couldn't find any info on how to use the command.

File: pages/linux/dmidecode.md
Comment (marchersimon) [inline pages/linux/dmidecode.md:5]:
```suggestion
> More information: <https://manned.org/dmidecode>.
```
Again no info on the official site.

File: pages/linux/do-release-upgrade.md
Comment (marchersimon) [inline pages/linux/do-release-upgrade.md:4]: Not sure about this one. It's well explained, but only one of the two flags are covered.

File: pages/linux/dos2unix.md
Comment (marchersimon) [inline pages/linux/dos2unix.md:5]:
```suggestion
> More information: <https://manned.org/dos2unix>.
```

File: pages/linux/dpkg-query.md
Comment (marchersimon) [inline pages/linux/dpkg-query.md:4]:
```suggestion
> More information: <https://manpages.debian.org/latest/dpkg/dpkg-query.1.html>.
```

File: pages/linux/dunstify.md
Comment (marchersimon) [inline pages/linux/dunstify.md:5]:
```suggestion
> More information: <https://wiki.archlinux.org/title/Dunst#Dunstify>.
```
The official website doesn't even mention `dunstify` as far as I have seen. The only useful documentation was on the Arch wiki.

**Review by marchersimon — COMMENTED** [review]
Everything from `chattr` to `dunstify` for now.

**Comment by Stig124** [comment]
A PR is comming for a nice modification I made to the script

File: pages/linux/export.md
Comment (marchersimon) [inline pages/linux/export.md:4]:
```suggestion
> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-export>.
```
Same as above

**Review by marchersimon — COMMENTED** [review]
`e2fsck` to `firejail`

File: pages/linux/flatpak.md
Comment (marchersimon) [inline pages/linux/flatpak.md:4]:
```suggestion
> More information: <https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak>.
```

File: pages/linux/gedit.md
Comment (marchersimon) [inline pages/linux/gedit.md:4]:
```suggestion
> More information: <https://help.gnome.org/users/gedit/stable/>.
```

File: pages/linux/guix-package.md
Comment (marchersimon) [inline pages/linux/guix-package.md:4]:
```suggestion
> More information: <https://guix.gnu.org/manual/html_node/Invoking-guix-package.html>.
```

**Review by marchersimon — COMMENTED** [review]
And the rest

**Comment by marchersimon** [comment]
@Stig124 are you done with this PR? If so, please mark it as `Ready for review` and have a look at my suggestions.

**Comment by Stig124** [comment]
> @Stig124 are you done with this PR? If so, please mark it as `Ready for review` and have a look at my suggestions.

I haven't had time to do anything, sorry (exam...), will try to do something to get that PR forward asap

**Comment by marchersimon** [comment]
No worries, take your time. You've already done some amazing work so far.

**Review by Waples — APPROVED** [review]

**Comment by Stig124** [comment]
Yes, will take time to do that in no time

**Comment by marchersimon** [comment]
@Stig124 in case you don't have the time for it just tell us and we take over from there.

**Comment by Stig124** [comment]
> @Stig124 in case you don't have the time for it just tell us and we take over from there.

Yes please, sorry for not having been able to finish that PR @marchersimon

**Comment by marchersimon** [comment]
No problem at all. Thanks for the work so far.

**Comment by CleanMachine1** [comment]
Ok, my review was requested so here it is:

GEdit, flatpak need fixing and if possible fsck

**Comment by marchersimon** [comment]
@CleanMachine1 I've already made suggestions for GEdit and flatpak. Are those links okay? And I don't see whats wrong with fsck.

**Comment by CleanMachine1** [comment]
Yes. However I have 1 more thing to mention. #6062 needs to have ALL ticked off. I am sure someone here can use some wizardry to auto do all of them here using a replace tool or whatever

**Comment by marchersimon** [comment]
Any thoughts on my suggestions above? If not I'll commit them.

**Review by marchersimon — APPROVED** [review]
All suggestions are applied and all translated pages are in sync. I think this PR is ready to be merged.

File: pages/linux/fdisk.md
Comment (CleanMachine1) [inline pages/linux/fdisk.md:5]:
```suggestion
> More information: <https://www.redhat.com/sysadmin/partitions-fdisk>.
```

Comment (CleanMachine1) [reply pages/linux/fdisk.md:5]: the fdisk manual really provides little actual information, and is just a really complicated guide, it was tldr

Comment (marchersimon) [reply pages/linux/fdisk.md:5]: Well, at least it shows all of the command line options. Personally I'd prefer to get an overview in the man page and then look if I find any useful tutorials

Comment (CleanMachine1) [reply pages/linux/fdisk.md:5]: Sometimes I wonder whether we should have a man page link and a more help link

Comment (marchersimon) [reply pages/linux/fdisk.md:5]: I think we discussed that in depth in #5510. Generally and especially in this case I prefer the man page, since I don't like just copy-pasting commads from blogs without knowing what the command can do else. The man page gives you an overview on what the command can do and then you can search for tutorials for those flags if you want to.

Comment (CleanMachine1) [reply pages/linux/fdisk.md:5]: I wouldn't ever suggest a random page from sketchywebpage.com, its from redhat.com and its one of their guides

Comment (marchersimon) [reply pages/linux/fdisk.md:5]: If there are certain things you have to pay attention to, shouldn't we include it in the page itself insead of linking to a page which shows how to solve it?

Comment (CleanMachine1) [reply pages/linux/fdisk.md:5]: I have another thought on it, this page is on the Linux category.

Most, if not all. distros have a man page, making this just a duplicate of what everyone using this command has in their offline terminal already

Comment (SethFalco) [reply pages/linux/fdisk.md:5]: > Most, if not all. distros have a man page, making this just a duplicate of what everyone using this command has in their offline terminal already

As far as I understand, the goal of tldr is just to provide an alternate way to consume documentation. The entire point is that it "duplicates" the information, and that's 100% fine.

I think in this case, manned is better for supplementary information:
* it provides more information (often first party)
* man pages will be kept up-to-date, articles like this usually aren't revised after publication
* it's more consistent with other pages (lots of manned links!)
* man pages are standardized (in the same ways tldr pages are standardized) so it's immediately familiar

I'm not against linking to external sources like articles, but I think we should prefer man pages, the link to the repository, or even the README over arbitrary articles regardless of reputability.

**Comment by CleanMachine1** [comment]
Cant review on mobile from what I see but this has my approval

**Comment by marchersimon** [comment]
It's possible to review by clicking on the `151 files changed on the top`.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**
